test/e2e/suites/e2e/e2e_test.go

```diff
@@ -161,11 +161,13 @@ var _ = Describe("e2e tests", func() {
 
 			filterNone := ports.ListOpts{}
 			plist1, err := shared.DumpOpenStackPorts(e2eCtx, filterNone)
+
 			Expect(err).To(BeNil())
 			Expect(plist1).NotTo(BeNil())
 
 			filterOne := ports.ListOpts{Description: "primary"}
 			plist2, err := shared.DumpOpenStackPorts(e2eCtx, filterOne)
+
```

Comment:
Adding new line here is out of concern of this PR. Please remove this new line.

Reply:
Done, thanks.